app/views/filters/_form.html.erb

```diff
@@ -56,7 +63,7 @@
     <% if show_location_tab? %>
       <div class="tab-pane" id="locations">
         <% collection = User.current.locations.empty? ? Location : User.current.locations %>
-        <%= multiple_checkboxes f, :locations, f.object, collection, :label => _("Locations") %>
+        <%= multiple_checkboxes f, :locations, f.object, collection, {}, :label => _("Locations") %>
```

Comment:
This is quite odd, and a bug that I'm not sure if it happens in develop, but I only see the top level taxonomies under these tabs. E.g: My setup has Lobacorp and Lobacorp/House as orgs, and Spain, Spain/Madrid as locations, yet..

![screenshot from 2016-08-25 11-22-51](https://cloud.githubusercontent.com/assets/598891/17963545/5acbf080-6ab6-11e6-809f-8f9bdebad77f.png)

It happens with both organizations and locations